docs/language-basics/built-in-types.md

```diff
@@ -27,9 +27,13 @@ Built in types are related to each other according to the rules described in [ty
 | `uint8`      | `0` to `255 `                                                                                                | Unsigned 8-bit integer                    |
 | `float32`    | <code> ±1.5 x 10<sup>45</sup></code> to <code>±3.4 x 10<sup>38</sup></code>                                  | A 32 bit floating point number            |
 | `float64`    | <code>±5.0 × 10<sup>−324</sup></code> to <code>±1.7 × 10<sup>308</sup></code>                                | A 64 bit floating point number            |
-| `decimal`    |                                                                                                              | A decimal number                          |
+| `decimal`    | [\*](#numeric-supertypes)                                                                                    | A decimal number                          |
 | `decimal128` | 34 decimal digits with an exponent range from `-6143` to `6144`                                              | A 128 bit decimal number                  |
 
+### Numeric supertypes
+
+`numeric`, `integer`, `float` and `decimal` are types that represent any possible number in their category. For some emit targets, `BigInt` or `BigDecimal` might be an analogous type satisfying the TypeSpec types `integer` and `decimal` respectively. For other targets where the language, serialization format, or protocol does not support an analogous type, emitters may decide on a policy for emitting the numeric supertypes. This might involve picking the closest analogous type or reporting an error when they are encountered, depending on the situation.
```

Comment:
```suggestion
`numeric`, `integer`, `float` and `decimal` are types that represent any possible number in their category. For some emit targets, `BigInt` or `BigDecimal` might be an analogous type satisfying the TypeSpec types `integer` and `decimal` respectively. For other targets where the language, serialization format, or protocol does not support an analogous type, emitters may decide on a policy for emitting the numeric supertypes. This might involve picking the closest analogous type and reporting a warning when they are encountered.
```